go.mod

```diff
@@ -10,5 +10,5 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.3.0
-	golang.org/x/sys v0.0.0-20210326220804-49726bf1d181 // indirect
+	modernc.org/sqlite v1.14.3
```

Comment:
In Focalboard we're using `github.com/mattn/go-sqlite3`, which seems to be the most widely used driver. Is there any reason to favor this one instead?

Reply:
I word: `CGO` :)